### ISSUE TITLE
Native crash during device support check on x86

### ISSUE THREAD

**Issue by ghost** [card-io/card.io-Android-SDK#193]
Device: motorola razr i, Android 4.4.2 (x86)
Lib version: 5.4.0 and later (perhaps, older versions)

Got crash during call CardIOActivity.canReadCardWithCamera() (called from onAttachedToWindow of button for running CardIOActivity).

Crash in CardScanner.java: 
    Log.d(Util.PUBLIC_LOG_TAG, "    nUseNeon(): " + nUseNeon());

Log:
    I/card.io: Checking hardware support...
    I/card.io: card.io 5.4.0 06/15/2016 15:23:04 -0500
    D/card.io: Loaded card.io decider library.

Tried to call the method from onPostCreate in MainActivity, the same.
Perhaps it is important to notice that I use own native libraries compiled with clang

**Comment by lkorth** [comment]
Have you tried with the latest version (5.5.1)? If not please switch to the latest version and post the log from the crash.

**Comment by ghost** [comment]
Yes, I've tried. The log is the same for three latest versions. It has to be noticed that sample works correctly. I suppose the problem in compilation issues. I'll try to recompile sample with clang or with some/our native code.

**Comment by lkorth** [comment]
There doesn't appear to be any stacktrace or crash information in the log that you posted.
